src/spec-node/collectionCommonUtils/outdatedCommandImpl.ts

```diff
@@ -0,0 +1,477 @@
+import * as jsonc from 'jsonc-parser';
+import * as os from 'os';
+import * as path from 'path';
+import * as semver from 'semver';
+import * as crypto from 'crypto';
+
+import { Log, LogLevel, mapLogLevel, nullLog } from '../../spec-utils/log';
+import { OutdatedArgs } from './outdated';
+import { CLIHost, getCLIHost } from '../../spec-common/cliHost';
+import { URI } from 'vscode-uri';
+import { loadNativeModule } from '../../spec-common/commonUtils';
+import textTable from 'text-table';
+import { ContainerError } from '../../spec-common/errors';
+import { getDevContainerConfigPathIn, getDefaultDevContainerConfigPath } from '../../spec-configuration/configurationCommonUtils';
+import { fetchOCIFeature, getFeatureIdWithoutVersion, tryGetOCIFeatureSet } from '../../spec-configuration/containerFeaturesOCI';
+import { getPackageConfig } from '../../spec-utils/product';
+import { workspaceFromPath } from '../../spec-utils/workspaces';
+import { readDevContainerConfigFile } from '../configContainer';
+import { createLog, createDockerParams } from '../devContainers';
+import { uriToFsPath, getCacheFolder, DockerResolverParameters, getDockerfilePath } from '../utils';
+import { DevContainerConfig, DevContainerFromDockerfileConfig, getDockerComposeFilePaths } from '../../spec-configuration/configuration';
+import { CommonParams, ManifestContainer, getRef, getVersionsStrictSorted } from '../../spec-configuration/containerCollectionsOCI';
+import { DockerCLIParameters } from '../../spec-shutdown/dockerUtils';
+import { request } from '../../spec-utils/httpRequest';
+import { readDockerComposeConfig, getBuildInfoForService } from '../dockerCompose';
+import { extractDockerfile, findBaseImages } from '../dockerfileUtils';
+import { ContainerFeatureInternalParams, userFeaturesToArray, getFeatureIdType, DEVCONTAINER_FEATURE_FILE_NAME, Feature } from '../../spec-configuration/containerFeaturesConfiguration';
+import { readLockfile } from '../../spec-configuration/lockfile';
+
+export interface OutdatedFeatures {
+	'features': {
+		[key: string]: {
+			'current': string;
```

Comment:
I also see 'wantedMajor' and 'latestMajor' in the JSON output, are these missing here or shouldn't make it to the output?

Reply:
Missed that, updated